docs/blog/flip-card.md

```diff
@@ -0,0 +1,32 @@
+---
+slug: flipCard
+title: Flip Card
+---
+
+Flip card component is a component that allows you to display different content depending on whether the card is flipped or not. It can be especially useful when you do not want to display some data immediately after entering the screen (e.g. secure data) and only after fulfilling a certain condition or performing an action.
+
+import FlipCard from '@site/static/examples/FlipCard';
+import FlipCardSrc from '!!raw-loader!@site/static/examples/FlipCard';
+import ExampleVideo from '@site/src/components/ExampleVideo';
+
+<InteractiveExample src={FlipCardSrc} component={<FlipCard />} />
+
+For storing information about whether the card is flipped or not we use [shared value](/docs/fundamentals/glossary#shared-value) with the `useSharedValue` hook. Using shared values helps to prevent unnecessary re-renders.
+
+<CollapsibleCode src={FlipCardSrc} showLines={[111,111]} />
+
+This allows us to [interpolate](/docs/utilities/interpolate) values between 0-180 and 180-360 degrees, depending on whether the card is flipped or not. In addition, we use [withTiming](/docs/animations/withTiming) util which makes our animation smooth.
+
+<CollapsibleCode src={FlipCardSrc} showLines={[56,58]} />
+
+<ExampleVideo 
+    sources={{ 
+        android: "/react-native-reanimated/recordings/examples/flip_card_android.mov",
```

Comment:
the phone used in this example is not the same as in other examples (Pixel 4)

Reply:
followup - the video is not cut properly
<img width="561" alt="image" src="https://github.com/software-mansion/react-native-reanimated/assets/59940332/75d8f0ae-5f15-45c1-84aa-1ea36c136ceb">